### PR TITLE
DOMA-2922 fix table scroll config & made the column width the same for any screen

### PR DESCRIPTION
**Pull request by Alllex202**

### CODE REVIEW

File: apps/condo/domains/common/utils/tables.utils.ts [hunk 1]
```diff
@@ -388,9 +388,7 @@ type TableScrollProps = RcTableProps['scroll'] & { scrollToFirstRowOnChange?: bo
 export const getTableScrollConfig = (isSmall: boolean): TableScrollProps => {
     const props: TableScrollProps = {}
 
-    if (isSmall) {
-        props.x = true
-    }
+    props.x = true
```
Comment (pahaz) [inline apps/condo/domains/common/utils/tables.utils.ts:391]: probably you need to remove isSamll argument

File: apps/condo/domains/common/utils/tables.utils.ts [hunk 2]
```diff
@@ -388,9 +388,7 @@ type TableScrollProps = RcTableProps['scroll'] & { scrollToFirstRowOnChange?: bo
 export const getTableScrollConfig = (isSmall: boolean): TableScrollProps => {
```
Comment (pahaz) [inline apps/condo/domains/common/utils/tables.utils.ts:388]: this function looks strange ... can we remove it?

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/common/utils/tables.utils.ts (hunk 2 above)
Comment (Alllex202) [reply apps/condo/domains/common/utils/tables.utils.ts:388]: Used all over the place, but I find it a useless function

**Review by nomerdvadcatpyat — APPROVED** [review]

File: apps/condo/domains/common/utils/tables.utils.ts [hunk 3]
```diff
@@ -388,9 +388,7 @@ type TableScrollProps = RcTableProps['scroll'] & { scrollToFirstRowOnChange?: bo
...
     const props: TableScrollProps = {}
 
-    if (isSmall) {
-        props.x = true
-    }
+    props.x = true
 
     return props
 }
```
[the remark this answers is not in the document]
Comment (Alllex202) [reply apps/condo/domains/common/utils/tables.utils.ts]: removed it

**Pull request merged**